src/poliastro/earth/__init__.py

```diff
@@ -0,0 +1,153 @@
+
+        """
+
+        if orbit.attractor.name != "Earth":
+            raise ValueError("The attractor must be Earth")
+
+        self._orbit = orbit  # type: Orbit
+
+    @property
+    def orbit(self):
+        """Orbit of the EarthSatellite. """
+        return self._orbit
+
+    @classmethod
+    @u.quantity_input(r=u.m, v=u.m / u.s)
+    def from_vectors(cls, r, v):
+        """Return EarthSatellite with the `Orbit` from position and velocity vectors.
+
+        Parameters
+        ----------
+        r : ~astropy.units.Quantity
+            Position vector wrt attractor center.
+        v : ~astropy.units.Quantity
+            Velocity vector.
+
+        Returns
+        -------
+        EarthSatellite
+            New EarthSatellite with the'Orbit' position and velocity vectors.
+
+
+        """
+
+        orbit = Orbit.from_vectors(Earth, r, v)
+        return cls(orbit)
+
+    @classmethod
+    @u.quantity_input(a=u.m, ecc=u.one, inc=u.rad, raan=u.rad, argp=u.rad, nu=u.rad)
+    def from_classical(
+        cls, a, ecc, inc, raan, argp, nu,
+    ):
+        """Return EarthSatellite with the 'Orbit' from classical orbital elements.
+
+        Parameters
+        ----------
+        a : ~astropy.units.Quantity
+            Semi-major axis.
+        ecc : ~astropy.units.Quantity
+            Eccentricity.
+        inc : ~astropy.units.Quantity
+            Inclination
+        raan : ~astropy.units.Quantity
+            Right ascension of the ascending node.
+        argp : ~astropy.units.Quantity
+            Argument of the pericenter.
+        nu : ~astropy.units.Quantity
+            True anomaly.
+
+        Returns
+        -------
+        EarthSatellite
+            New EarthSatellite with the 'Orbit' from classical orbital elements.
+
+        """
+
+        orbit = Orbit.from_classical(Earth, a, ecc, inc, raan, argp, nu)
+        return cls(orbit)
+
+    def propagate(self, tof, atmosphere=None, gravity=None, A_over_m=None, *args):
+        """Propagates an 'EarthSatellite Orbit' at a specified time.
+
+        If value is true anomaly, propagate orbit to this anomaly and return the result.
+        Otherwise, if time is provided, propagate this `EarthSatellite Orbit` some `time` and return the result.
+
+        Parameters
+        ----------
+
+        tof : ~astropy.units.Quantity, ~astropy.time.Time, ~astropy.time.TimeDelta
+            Scalar time to propagate.
+        atmosphere:
+            a callable model from poliastro.atmosphere
+        gravity: EarthGravity
+            There are two possible values, SPHERICAL and J2. Only J2 is implemented at the moment. Default value is None.
+        A_over_m: float
+            frontal area/mass of the spacecraft (km^2/kg)
+        *args:
+            parameters used in perturbation models.
+
+        Returns
+        -------
+        EarthSatellite
+            A new EarthSatellite with the propagated Orbit
+
+        """
+
+        ad_kwargs: Dict[object, dict] = {}
```

Comment:
I was not super convinced about adding PEP 526 variable annotations... But what the hell, we're in 2020 and poliastro requires Python 3.6 already :)